src/styles/SearchBar.css

```
@@ -0,0 +1,28 @@
+.input-search-bar {
+    width: 668px;
+    height: 36px;
+    border-radius: 99px;
+    border: none;
+    background: url(../utils/images/lupa.svg) 15px no-repeat #EDEDF0;
+    padding-left: 45px;
+}
+
+.button-search-bar {
+    width: 88px;
+    height: 36px;
+    border-radius: 99px;
+    background: #FFFFFF;
+    border: 1px solid #9DC2FF;
+    color: #2264D1;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 143%;
+    margin-left: 15px;
+}
+
+.container-search-bar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+}
```

Aqui, só para o ESlint não ficar reclamando, o que acha de colocar mais uma linha? 
```suggestion
}

```

---

Já estou ajustando, vlw   ;)